macOS/LocalPackages/DataBrokerProtection-macOS/Sources/DataBrokerProtection-macOS/BackgroundAgent/DataBrokerProtectionAgentManager.swift

```diff
@@ -307,6 +311,18 @@ extension DataBrokerProtectionAgentManager: DataBrokerProtectionBackgroundActivi
     }
 }
 
+extension DataBrokerProtectionAgentManager: DataBrokerProtectionQueueManagerDelegate {
+
+    public func queueManagerWillEnqueueOperations(_ queueManager: DataBrokerProtectionQueueManager) {
+        Task {
+            do {
+                try await brokerUpdater.checkForUpdates()
+            }
+        }
+    }
```

Comment:
This doesn’t address the concern that checkForUpdates() is non-blocking, but now it isn’t part of the queue manager anymore.

Reply:
What's the current plan there?

Reply:
We discussed this over MM and the plan is to do it in a follow up